### PR TITLE
Added REST API features for Poppo UI.

### PR DESCRIPTION
**Pull request by JoeyyT**
Will update this more often.

### CODE REVIEW

**Comment by Sieberkev** [comment]
You do not need to merge the whole HTML/CSS/JS/... code to the `gh-pages`, you can add your existing repository to the `.gitmodules` file so it will make a directory/link to it (only the readme and index.html file needs to be changed to link to that directory).

So if you want to add/update stuff on your GUI, you can do so on your own repository without having to wait on PR merges here :)

File: TTL55/scripts/app.js [hunk 1]
```diff
@@ -0,0 +1,618 @@
...
+	$(".pokemonListDiv").show();
+	$(".editPokemon").hide();
+	$("#close").show();
+	$("#return").hide();
+}
+
+
+
+function getUrl() {
+	return "http://localhost:8080/api/bot/default/";
```
Comment (Peyphour) [inline TTL55/scripts/app.js]: Here, if my bot is somewhere else or has a different name ?

Comment (JoeyyT) [inline TTL55/scripts/app.js]: Yeah outdated getURL(); its fixed
One sec

**Comment by Peyphour** [comment]
Is it possible de fetch the bot names from `/api/bots` instead of asking the user to input them ?

**Comment by Sieberkev** [comment]
Merged as a git submodule in #1215

**Pull request closed without merging**